components/SearchBox.vue

```diff
@@ -0,0 +1,407 @@
+<template>
+  <div id="search" class="search-container" :class="{active: search_active}">
+    <div class="search-icon-container">
+      <div class="search-icon dark small">
+        <div class="mug-top" />
+        <div class="mug-sides" />
+        <div class="mug-bottom" />
+        <div class="mug-handle">
+          <div class="mug-handle-inner" />
+        </div>
+      </div>
+    </div>
+
+    <input
+      v-model="query"
+      type="text"
+      class="form-control search"
+      placeholder="Search beers, brewers, or styles"
+      aria-label="Search beers, brewers, or styles"
```

Comment:
```suggestion
      placeholder="Search beers, breweries, or styles"
      aria-label="Search beers, breweries, or styles"
```

Minor stylistic thing here since you probably want to search for Straight to Ale rather than Bob Gile (STA's head brewer)